lib/email_auth.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+#email_auth.py
+#
+#Combine standard login with option of OAuth2.0
+#
+from httplib2 import Http
+from oauth2client import file, client, tools
+from googleapiclient.discovery import build
+from lib.config_file import open_config
+import smtplib
+import base64
+from imaplib import IMAP4_SSL
+
+
+def email_authentication():
+   config = open_config()
+   email_config = config['email']
+   SCOPES = email_config["gmailUrl"]
+   username= email_config["username"]
+   mail = IMAP4_SSL(email_config['imapUrl'])
+   if "password" in email_config:
```

Comment:
this might be a gotcha since the password field is in the config template -- we should check that the password is not empty as well, since users might just have an empty password created by copying the template

Reply:
updated to consider empty field